docs/examples/index.md

```diff
@@ -1,278 +1,41 @@
-          <p>Efficient window functions for ranking, computing deltas, and tracking cumulative sums</p>
-        </div>
-      </div>
-    </a>
-  </div>
+    Generate and store millions of text embeddings in vector databases using distributed GPU processing and state-of-the-art models.
 
-</div>
-
-<style>
-.examples-grid {
-  display: grid;
-  grid-template-columns: repeat(3, 1fr);
-  gap: 0.5rem;
-  margin: 0.5rem 0;
-}
-
-.example-card {
-  border-radius: 12px;
-  overflow: hidden;
-  box-shadow: 0 4px 12px rgba(0, 0, 0, 0.1);
-  transition: transform 0.2s ease, box-shadow 0.2s ease;
-  background: white;
-}
-
-.example-card:hover {
-  transform: translateY(-4px);
-  box-shadow: 0 8px 24px rgba(0, 0, 0, 0.15);
-}
-
-.example-image {
-  position: relative;
-  height: 380px;
-  overflow: hidden;
-}
-
-.example-image img {
-  width: 100%;
-  height: 100%;
-  object-fit: cover;
-  transition: transform 0.3s ease;
-}
-
-.example-placeholder {
-  position: absolute;
-  top: 0;
-  left: 0;
-  width: 100%;
-  height: 100%;
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  font-size: 3rem;
-  color: white;
-  text-shadow: 0 2px 4px rgba(0, 0, 0, 0.3);
-}
+- [Generate Images with Stable Diffusion](image-generation.md)
 
-.example-image::after {
-  content: "";
-  position: absolute;
-  top: 0;
-  left: 0;
-  right: 0;
-  bottom: 0;
-  z-index: 1;
-  pointer-events: none;
-}
+    Open Source image generation model on your own GPUs using Daft UDFs.
 
-.example-card:hover .example-image img {
-  transform: scale(1.05);
-}
+- [Window Functions: The Great Chocolate Race](window-functions.md)
 
-.example-image-link {
-  display: block;
-  text-decoration: none;
-  color: inherit;
-  cursor: pointer;
-}
+    Explore how window functions can reduce complex joins and groupby's to just a few simple operations.
 
-.example-image-link:hover {
-  text-decoration: none;
-  color: inherit;
-}
+- [Running LLMs on the Red Pajamas Dataset](llms-red-pajamas.md)
 
-.example-overlay {
-  position: absolute;
-  top: 0;
-  right: 0;
-  bottom: 0;
-  left: 0;
-  background: linear-gradient(to top, rgba(0, 0, 0, 0.8) 12%, rgba(0, 0, 0, 0.4) 35%, rgba(0, 0, 0, 0) 65%);
-  color: white;
-  padding: 1.5rem 1rem 1rem;
-  display: flex;
-  flex-direction: column;
-  justify-content: flex-end;
-  z-index: 2;
-}
+    Perform similarity search on Stack Exchange questions using language models and embeddings.
 
-.example-overlay h3 {
-  margin: 0 0 0.5rem 0;
-  font-size: 1rem;
-  font-weight: 600;
-}
-
-.example-overlay p {
-  margin: 0;
-  font-size: 0.8rem;
-  opacity: 0.9;
-  line-height: 1.4;
-}
-
-.example-overlay-dark {
-  background: linear-gradient(to top, rgba(0, 0, 0, 0.9) 12%, rgba(0, 0, 0, 0.6) 35%, rgba(0, 0, 0, 0) 65%);
-}
-
-@media (max-width: 768px) {
-  .examples-grid {
-    grid-template-columns: 1fr;
-    gap: 1.5rem;
-  }
-
-  .example-image {
-    height: 180px;
-  }
-
-  .example-overlay {
-    padding: 1rem 0.75rem 0.75rem;
-  }
+</div>
```

Comment:
(Not in this PR, but it'll be great to add image and video examples here at some point. Hopefully soon.)